studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```diff
@@ -118,7 +118,7 @@ const FeedbackWidget = ({
           pathname: router.asPath,
         })
         setFeedback('')
-        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!' })
+        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!\n\nPlease note this channel is unmonitored, so you will not receive a reply.' })
```

Comment:
Maybe add a note that, if they need a human to reply, they should open a support ticket? Otherwise probably not super clear for some of our users what to do with that information

I'm not good with words, but something like

> Thank you! Please note that we don't reply to feedback. If you're looking for help or a response, please consider opening a support ticket.

Reply:
Updated it 👍 
![image](https://github.com/supabase/supabase/assets/99693443/32e5c2c9-2529-4ceb-83de-44380060ef9f)